components/automate-gateway/api/iam/v2/rules.proto

```diff
@@ -14,7 +15,52 @@ import "components/automate-grpc/protoc-gen-policy/api/annotations.proto";
 import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 
 service Rules {
+
+  /*
+  Creates a project rule
```

Comment:
Not to belabor the issue 😁  but because there was a lot of talk about this it may not have been clear where this ended up. This phrase (which goes _only_ in the navigation panel), as confirmed by @kagarmoe, and as used in all of the non-auth API docs as well as some of the auth API docs, is supposed to be singular. (Either way it will be "inconsistent" because e.g., tokens uses plural while users uses singular. 🤷‍♂ )